Bottom: 1a650c8d9d82e360ef6f64688e05bf74506fabe9
Top:    554976292801198ea54943729b8babca57a70464
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-05 20:31:20 +0000

Refresh of patch

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 5a60464..9740e9b 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -418,8 +418,8 @@ step_absr_fragment (step_file *step,
   /* Emit references to the styled and over_ridden styled items */
   step_mechanical_design_geometric_presentation_representation (step, "", styled_item_list, geometric_representation_context_identifier);
 
-  if (shape_representation != NULL)
-    *shape_representation = shape_representation_identifier;
+//  if (shape_representation != NULL)
+//    *shape_representation = shape_representation_identifier;
 
   if (shape_definition_representation != NULL)
     *shape_definition_representation = shape_definition_representation_identifier;
@@ -612,7 +612,7 @@ object3d_list_export_to_step_assy (GList *objects, const char *filename)
                       NULL, //styled_item_identifiers,
                       assy_geometric_representation_context_identifier,
                       assy_product_definition_shape_identifier,
-                      NULL /* shape_representation */,
+//                      NULL /* shape_representation */,
                       NULL /* shape_definition_representation */,
                       &origin_axis); /* This is set, not NULL so is taken as an input */
